docs/data-cloud/aspire-integration.md

```diff
@@ -0,0 +1,344 @@
+
+Traditionally, connecting a .NET MAUI app to local web services requires significant manual configuration. You need to:
+
+- Use different URLs for different platforms (localhost, 10.0.2.2, etc.)
+- Configure network security settings for Android
+- Set up Apple Transport Security (ATS) for iOS
+- Handle certificate validation for HTTPS
+- Manually manage service URLs in your code
+
+For more information about the traditional approach, see [Connect to local web services](local-web-services.md).
+
+With Aspire integration, these complexities are handled automatically, allowing you to focus on building your application instead of configuring network access.
+
+## Prerequisites
+
+To use Aspire with .NET MAUI, you need:
+
+- [.NET 10 SDK](https://dotnet.microsoft.com/download/dotnet/10.0) or later
+- [Aspire workload](/dotnet/aspire/fundamentals/setup-tooling)
+- A .NET MAUI app targeting .NET 10 or later
+- One or more ASP.NET Core web services
+
+To install the Aspire workload, run:
+
+```dotnetcli
+dotnet workload install aspire
+```
+
+## Getting started
+
+Setting up Aspire integration with your .NET MAUI application involves adding two key projects to your solution:
+
+1. **MAUI Service Defaults project**: Provides default configuration for your MAUI app
+2. **App Host project**: Orchestrates your application services and handles service discovery
+
+### Create a MAUI Service Defaults project
+
+The MAUI Service Defaults project contains shared configuration that your MAUI application will use to connect to services. Create this project in your solution directory:
+
+```dotnetcli
+dotnet new maui-aspire-servicedefaults -n YourApp.MauiServiceDefaults
+```
+
+This project includes:
+
+- Service discovery configuration
+- Default resilience patterns
+- Telemetry setup
+- Platform-specific networking configuration
+
+Add a reference to this project from your .NET MAUI app project:
+
+```dotnetcli
+dotnet add YourMauiApp.csproj reference YourApp.MauiServiceDefaults/YourApp.MauiServiceDefaults.csproj
+```
+
+### Create an App Host project
+
+The App Host project orchestrates all your application services, including your MAUI app and any backend services. Create this project in your solution directory:
+
+```dotnetcli
+dotnet new aspire-apphost -n YourApp.AppHost
+```
+
+Add references to your MAUI app and any web service projects:
+
+```dotnetcli
+dotnet add YourApp.AppHost.csproj reference YourMauiApp/YourMauiApp.csproj
+dotnet add YourApp.AppHost.csproj reference YourWebService/YourWebService.csproj
+```
+
+### Configure the App Host
+
+In your App Host project's `Program.cs`, register your MAUI app and web services:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+// Register your web service
+var apiService = builder.AddProject<Projects.YourWebService>("apiservice");
+
+// Register your MAUI app with a reference to the API service
+builder.AddProject<Projects.YourMauiApp>("mauiapp")
+    .WithReference(apiService);
```

Comment:
This is not correct, be sure to check https://github.com/dotnet/aspire/blob/main/playground/AspireWithMaui/AspireWithMaui.AppHost/AppHost.cs